releases/release-1.6/README.md

```diff
@@ -0,0 +1,52 @@
+# Kubeflow 1.6
+
+### Links
+
+- [Release Team](release-team.md)
+- [Calendar](https://arrik.to/kf-release-team-cal)
+- [Meeting Notes](https://arrik.to/kf-release-team-notes)
+- [Recordings](https://arrik.to/kf-release-team-recordings)
+- [Retrospective Document](https://bit.ly/kf-release-1-6-retro)
+- Project Board
+- Contact
+  - [#release](https://app.slack.com/client/T7QLHSH6U/C9V2WT2KV) on slack
+  - [@release-team](https://github.com/orgs/kubeflow/teams/release-team) on GitHub
+
+### TL;DR
+
+The 1.6 release cycle is proposed as follows
+
+- **Wednesday, Mar 16th 2022**: Week 1 - Release Cycle Begins
```

Comment:
I assume you mean May, not march

Reply:
March is correct here. The proposal is that the 1.6 release process started a week after the 1.5 was released. All other deadlines here are based on the fact that the release cycle began on March 16th.